webpack-test/cases/json/import-by-name/test.filter.js

```diff
@@ -1,4 +1,4 @@
 
-module.exports = () => {return "https://github.com/web-infra-dev/rspack/issues/4323"}
```

Comment:
want to make sure json tree shaking generate valid result.

Reply:
Try not to modify the webpack-test, so we can track the different behavior, if only for valid the result, you can create copy the test into `packages/rspack/tests`

Reply:
Done